packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -253,6 +231,11 @@ class SystemRulesTable extends React.Component {
                 title: (compliant ? <CheckCircleIcon className='ins-u-passed' /> :
                     <ExclamationCircleIcon className='ins-u-failed' />),
                 original: compliant
+            },
+            {
+                title: (remediationAvailable ? <CheckIcon style={ { color: 'var(--pf-global--success-color--200)' } }/> :
```

Comment:
Can you make this into a style in `.scss` file? Inline styles are not really good, you can add `ins-c-compliance-system-rule-check` className to this component and style it in your stylesheet file.